TUM Campus App/ViewControllerProvider.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ViewControllerProvider.swift
+//  TUM Campus App
+//
+//  Created by Robert on 06.10.17.
+//  Copyright © 2017 LS1 TUM. All rights reserved.
+//
+
+import UIKit
+
+class ViewControllerProvider {
+    static private(set) var loginNavigationViewController: UINavigationController? = {
```

Comment:
Do you absolutely need it to be of type `UINavigationController`? Otherwise, change it to `UIViewController` so that it isn't optional and then you can get rid of the guard with break inside a switch